packages/webapp/src/pages/Connection/List.tsx

```diff
@@ -133,45 +153,64 @@ export default function ConnectionList() {
-                            <input
-                                id="search"
-                                name="search"
-                                type="text"
+                    <div className="flex justify-end w-full text-[12px] text-white">
+                        {connections.length} connections{' '}
+                        {errorNotifications && (
+                            <span className="flex items-center ml-1">
+                                ({errorNotifications} errored)<span className="ml-1 bg-red-base h-1.5 w-1.5 rounded-full"></span>
+                            </span>
+                        )}
+                    </div>
+                    <div className="flex gap-2 relative my-3">
+                        <div className="flex-grow">
+                            <Input
+                                before={<MagnifyingGlassIcon className="w-4" />}
                                 placeholder="Search by ID"
-                                className="border-border-gray bg-active-gray indent-8 text-white block w-full appearance-none rounded-md border px-3 py-2 text-sm placeholder-gray-400 shadow-sm focus:outline-none"
+                                className="border-active-gray"
                                 onChange={handleInputChange}
                                 onKeyUp={handleInputChange}
                             />
                         </div>
-                        <select
-                            id="integration"
-                            name="integration"
-                            className="ml-4 bg-active-gray border-border-gray text-text-light-gray block appearance-none py-1 rounded-md text-sm shadow-sm"
-                            onChange={handleIntegrationChange}
-                            value={selectedIntegration}
-                        >
-                            <option key="all" value="_ALL">
-                                All Integrations
-                            </option>
-                            {providers.map((integration: string) => (
-                                <option key={integration} value={integration}>
-                                    {integration}
-                                </option>
-                            ))}
-                        </select>
+                        <div className="flex">
+                            <MultiSelect
+                                label="Integrations"
+                                options={providers.map((integration: string) => {
+                                    return { name: integration, value: integration };
+                                })}
+                                selected={selectedIntegration}
+                                defaultSelect={defaultFilter}
+                                onChange={handleIntegrationChange}
+                                all
+                            />
+                            <MultiSelect
```

Comment:
Nice that you could reuse that without change 👌🏻

Reply:
Ideally I wanted to use the multi select with search but that was very much tied to the ES implementation so left it alone

Reply:
yeah search is rarely portable, unless you do everything in the browser or you had a function parameter but then the component is much more complicated to use